server/handlers.go

```diff
@@ -0,0 +1,283 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/eth"
+)
+
+func logAndRespondWithError(w http.ResponseWriter, errMsg string, code int) {
+	glog.Error(errMsg)
```

Comment:
Could be cool to include the HTTP code along with the msg, wdyt?

Reply:
Yeah but we don't write the status code to the log, right? - I was hoping we could log the status code. wdyt?

Reply:
Added the status code [here](https://github.com/livepeer/go-livepeer/pull/658/files#diff-105fedf355a9fcad82982745cef24f7aR24)